### DISCUSSION TITLE
Wrong School name while signing up github edu with email.

### DISCUSSION THREAD

**Discussion by yoni13** [community/community#57081] [GitHub Education] [answered]
### Hi everyone,

I am trying to help my friend sign up for GitHub education benefits, He got denied after 1 second.
There's a problem that I and my friend cannot fix ourselves.

> The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.

Our school email(stu.sggs.hc.edu.tw), has been flagged as "National Hsinchu Girls' Senior High School".
But Our school's name is "曙光女子高級中學 Shu Guang Girls’ Senior High School", and emails end with "sggs.hc.edu.tw".

Hoping this gets fixed soon.

**Comment by karimdevelops** [comment]
Hey, maybe you can find your solution here: #53463 
Let me know if it works

**Reply by yoni13** [reply]
Nah, that's not gonna work if the school name on the form is wrong
![image](https://github.com/community/community/assets/76271912/d229b9bb-0e43-48de-859f-2f3f53077cc2)

**Reply by karimdevelops** [reply]
Ah, do you have an alternate? ID Card, or any other proof that you are a student? If so then submit that.
Let me know if it works

**Reply by yoni13** [reply]
Blocked because it is a new account, we will try again in a few days.

**Reply by karimdevelops** [reply]
I hope it will work

**Reply by yuhuam03** [reply]
You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

**Reply by yoni13** [reply]
Tried student id, official transcript with dates.
All of them denied.
I think I am not going to pass it unless the school name changes.

**Comment by samus-aran** [comment, marked as answer]
Hi @yoni13  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown **"My selected school has incorrect or incomplete information e.g. domains or campus location"**. 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍

**Reply by yoni13** [reply]
Thanks for replying two months later,we have submitted the support form and are waiting for a response.

**Reply by java-walker** [reply]
解决了吗
